bfdriver/tests/CMakeLists.txt

```diff
@@ -16,20 +16,44 @@
 # License along with this library; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 
-include(${CMAKE_INSTALL_PREFIX}/cmake/CMakeGlobal_Includes.txt)
+cmake_minimum_required(VERSION 3.6)
+project(bfdriver_test C CXX)
+include(${BF_FLAGS_DIR}/flags.cmake)
+include(CTest)
```

Comment:
Why not add a:

```
include(${BF_FLAGS_DIR}/test.cmake)
```

That sets up testing for you to reduce verbosity. Something like. 

```
cmake_minimum_required(VERSION 3.6)
project(bfdriver_test C CXX)

include(${BF_FLAGS_DIR}/test.cmake)
include(${BF_FLAGS_DIR}/flags.cmake)
```